spec/system/create_invite_spec.rb

```diff
@@ -0,0 +1,307 @@
+# frozen_string_literal: true
+
+describe "Creating Invites", type: :system do
+  fab!(:group)
+  fab!(:user) { Fabricate(:user, groups: [group]) }
+  fab!(:topic) { Fabricate(:post).topic }
+  let(:user_invited_pending_page) { PageObjects::Pages::UserInvitedPending.new }
+  let(:create_invite_modal) { PageObjects::Modals::CreateInvite.new }
+  let(:cdp) { PageObjects::CDP.new }
+
+  def open_invite_modal
+    find(".user-invite-buttons .btn", match: :first).click
+  end
+
+  def display_advanced_options
+    create_invite_modal.edit_options_link.click
+  end
+
+  before do
+    SiteSetting.invite_allowed_groups = "#{group.id}"
+    SiteSetting.invite_link_max_redemptions_limit_users = 7
+    SiteSetting.invite_link_max_redemptions_limit = 63
+    SiteSetting.invite_expiry_days = 3
+    sign_in(user)
+  end
+
+  before do
+    user_invited_pending_page.visit(user)
+    open_invite_modal
+  end
+
+  it "is possible to create an invite link without toggling the advanced options" do
+    cdp.allow_clipboard
+
+    create_invite_modal.save_button.click
+    create_invite_modal.copy_button.click
+
+    invite_link = create_invite_modal.invite_link_input.value
+    invite_key = invite_link.split("/").last
+
+    cdp.clipboard_has_text?(invite_link)
+
+    expect(create_invite_modal.link_limits_info_paragraph).to have_text(
+      "Link is valid for up to 7 users and expires in 3 days.",
+    )
+
+    create_invite_modal.close
+
+    invites = user_invited_pending_page.invites_list
```

Comment:
Can you check if it's better now? I can't get the invite ID from the invite modal after an invite has been created because it's not shown/rendered anywhere in the DOM, so I resorted to picking the first row from the invites list because the list is sorted in reverse chronological order.

Reply:
Why not add a `data-invite-id` to the invite table row?

Reply:
Wait scratch that, I see what you are saying now. What I was mostly meaning was that, instead of storing a reference to the capybara element in the code, you need to look it up every time you need the value. So your page object would become this:

```ruby
def invites_list_latest_invite
  Invite.new(find("#user-content .user-invite-list tbody tr:first-of-type"))
end
```

Then instead of this:

```ruby
invite = user_invited_pending_page.invites_list[0]
expect(invite).to be_link_type(key: invite_key, redemption_count: 0, max_redemption_count: 53)
expect(invite).to have_group(group)
expect(invite).to have_topic(topic)
expect(invite.expiry_date).to be_within(2.minutes).of(Time.zone.now + 90.days)
```

You have this:

```ruby
expect(user_invited_pending_page.invites_list_latest_invite).to be_link_type(key: invite_key, redemption_count: 0, max_redemption_count: 53)
expect(user_invited_pending_page.invites_list_latest_invite).to have_group(group)
expect(user_invited_pending_page.invites_list_latest_invite).to have_topic(topic)
expect(user_invited_pending_page.invites_list_latest_invite.expiry_date).to be_within(2.minutes).of(Time.zone.now + 90.days)
```

This is to avoid stale references of the capybara element, which can cause flakys.

Reply:
Ok, I see, I've updated the PR to use your suggestions.